src/main/java/org/tikv/common/TiSession.java

```diff
@@ -103,6 +120,22 @@ public TiSession(TiConfiguration conf) {
     warmUp();
     this.circuitBreaker = new CircuitBreakerImpl(conf);
     logger.info("TiSession initialized in " + conf.getKvMode() + " mode");
+    logger.info("Welcome to TiKV Java Client {}", getVersionInfo());
```

Comment:
It would print a welcome message when every new `TiSession` is created. Considering that an application can use multiple `TiSession`s, is there a better place to print this message?

Reply:
Done.